tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -131,7 +131,7 @@ stages:
               -e NIGHTLY_BUILD \
               -e BUILD_BUILDNUMBER \
               onnxruntimecpubuild \
-                bash -c " $(PythonManylinuxDir)/bin/python3 -m pip install /build/Release/dist/onnxruntime*.whl && /onnxruntime_src/tools/doc/builddoc.sh $(PythonManylinuxDir)/bin/ /onnxruntime_src /build Release " ;
+                bash -c " $(PythonManylinuxDir)/bin/python3 -m pip install /build/Release/dist/*.whl && /onnxruntime_src/tools/doc/builddoc.sh $(PythonManylinuxDir)/bin/ /onnxruntime_src /build Release " ;
```

Comment:
just to confirm. this change is needed because the nightly packages are named ort-nightly* ?

Reply:
Yes, I think so.

Reply:
Right, when pipeline variable ORT_NIGHTLY=0 (which is what we use to test, and what I did [here](https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=157886&view=results)) the wheel file name is onnxruntime*.whl. When ORT_NIGHTLY=1 the filename is ort-nightly*.whl.